src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBaker.java

```diff
@@ -1,24 +1,90 @@
 package com.minecolonies.coremod.colony.buildings;
 
+import com.google.common.collect.ImmutableMap;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.collect.ImmutableMap'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)